packages/utils/hooks/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@strapi/hooks",
+  "main": "./lib/index.js",
+  "version": "0.1.0",
```

Comment:
FYI right now we publish all packages in the same verison each time so this will directly get release as v4 if we don't move to independent versionning

Reply:
Sounds good to me 😊 . Thanks for the mention!